doc/spec/ground_truth_init_parameters.adoc

```diff
@@ -5,17 +5,17 @@ endif::[]
 = Ground truth initialization parameters
 
 All models can optionally consume `osi3::GroundTruth` via an initialization parameter called `OSMPGroundTruthInit`.
-Its purpose is to provide the model with a view of the static environment, for example the map, in OSI format.
+Its purpose is to provide the model with information that is constant throughout the simulation, for example the map in OSI format or the model_reference of a MovingObject.
```

Comment:
Let's discuss the wording for static/constant/moving/dynamic for changes in OSI3.8, as it also touches topics as in https://github.com/OpenSimulationInterface/open-simulation-interface/pull/768.

How about: `Its purpose is to provide the model with information that does not change throughout the simulation, for example the map in OSI format or the model_reference of a MovingObject.`

Reply:
I changed the sentence.

Reply:
I generally changed 'static'/'constant' to not 'changing'